pkg/tests/tasks/security/authorization/tcp_test.go

```diff
@@ -16,156 +16,195 @@ package authorization
 
 import (
 	"fmt"
-	"strings"
 	"testing"
-	"time"
 
-	"github.com/maistra/maistra-test-tool/pkg/examples"
-	"github.com/maistra/maistra-test-tool/pkg/util"
-	"github.com/maistra/maistra-test-tool/pkg/util/log"
+	"github.com/maistra/maistra-test-tool/pkg/app"
+	"github.com/maistra/maistra-test-tool/pkg/util/check/assert"
+	"github.com/maistra/maistra-test-tool/pkg/util/hack"
+	"github.com/maistra/maistra-test-tool/pkg/util/oc"
+	"github.com/maistra/maistra-test-tool/pkg/util/pod"
+	"github.com/maistra/maistra-test-tool/pkg/util/retry"
 	"github.com/maistra/maistra-test-tool/pkg/util/test"
 )
 
-func cleanupAuthorTCP() {
-	log.Log.Info("Cleanup")
-	util.KubeDeleteContents("foo", TCPDenyGETPolicy)
-	util.KubeDeleteContents("foo", TCPAllowGETPolicy)
-	util.KubeDeleteContents("foo", TCPAllowPolicy)
-	echo := examples.Echo{Namespace: "foo"}
-	echo.UninstallWithProxy()
-	sleep := examples.Sleep{Namespace: "foo"}
-	sleep.Uninstall()
-	time.Sleep(time.Duration(20) * time.Second)
+// TestAuthorizationTCPTraffic validates authorization polices for TCP traffic.
+func TestAuthorizationTCPTraffic(t *testing.T) {
+	test.NewTest(t).Id("T21").Groups(test.Full, test.InterOp).Run(func(t test.TestHelper) {
```

Comment:
groups can be defined as Full, InterOp because you have reference to test package

Reply:
Could be because are inside another package, but is ok tool to leave like test.Full, test.InterOp